libvirt/haproxy/haproxy_lnxocp01.cfg

```diff
@@ -151,32 +151,42 @@ frontend https-all
 #---------------------------------------------------------------------
 backend masters-00
     mode        tcp
+    option      httpchk GET /readyz HTTP/1.0
+    # Uncomment log-health-checks option only when in production or debugging, disabled for CI
+    # option    log-health-checks
     balance     roundrobin
-    server      bootstrap 192.168.126.10:6443 check
-    server      master1 192.168.126.11:6443 check
-    server      master2 192.168.126.12:6443 check
-    server      master3 192.168.126.13:6443 check
+    server      bootstrap 192.168.126.10:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master1 192.168.126.11:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master2 192.168.126.12:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master3 192.168.126.13:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
 backend masters-01
     mode        tcp
+    option      httpchk GET /readyz HTTP/1.0
+    # Uncomment log-health-checks option only when in production or debugging, disabled for CI
+    # option    log-health-checks
     balance     roundrobin
-    server      bootstrap 192.168.1.10:6443 check
-    server      master1 192.168.1.11:6443 check
-    server      master2 192.168.1.12:6443 check
-    server      master3 192.168.1.13:6443 check
+    server      bootstrap 192.168.1.10:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master1 192.168.1.11:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master2 192.168.1.12:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master3 192.168.1.13:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
 backend masters-02
     mode        tcp
+    option      httpchk GET /readyz HTTP/1.0
+    # Uncomment log-health-checks option only when in production or debugging, disabled for CI
+    # option    log-health-checks
     balance     roundrobin
-    server      bootstrap 192.168.2.10:6443 check
-    server      master1 192.168.2.11:6443 check
-    server      master2 192.168.2.12:6443 check
-    server      master3 192.168.2.13:6443 check
+    server      bootstrap 192.168.2.10:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master1 192.168.2.11:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master2 192.168.2.12:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
+    server      master3 192.168.2.13:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
 backend masters-03
     mode        tcp
+    
```

Comment:
missing the `option httpchk GET /readyz HTTP/1.0`

Reply:
Thank you @prb112